Core/tests/Unit/RequestWrapperTest.php

```diff
@@ -31,6 +31,7 @@
 use Google\Cloud\Core\RequestWrapper;
 use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Promise;
+use GuzzleHttp\Promise\Create;
```

Comment:
Woops, that was wrong. Change it back!
```suggestion
use GuzzleHttp\Promise\Create;
```